lib/frederick_api/v2/background_job.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module FrederickAPI
+  module V2
+    # V2 Frederick API async background job record, does not use a DB, just
+    # exists to provide a place to parse background job responses coming from
+    # API so that we can handle them properly.
+    class BackgroundJob < Resource
+      attr_accessor :meta
```

Comment:
`meta` is part of the json api spec so I would probably use `attr_accessor :response` instead since that looks like what you're tracking